ogr/services/pagure/pull_request.py

```diff
@@ -104,6 +104,10 @@ def created(self) -> datetime.datetime:
     def diff_url(self) -> str:
         return f"{self.url}#request_diff"
 
+    @property
+    def last_commit(self):
```

Comment:
Maybe a `head_commit` would fit better. (Last is a bit misleading because I don't know the direction...)

Reply:
changed